src/main/java/org/spongepowered/mod/SpongeMinecraftVersion.java

```diff
@@ -39,6 +40,11 @@ public SpongeMinecraftVersion(String name, int protocol) {
         this.protocol = protocol;
     }
 
+    public SpongeMinecraftVersion(ServerStatusResponse.MinecraftProtocolVersionIdentifier identifier) {
```

Comment:
Blood told me to throw it in there, he said it should be there. I can remove it though and let someone else add it.